src/app/components/story/BookList.js

```diff
@@ -11,39 +11,114 @@ import Book from './Book'
 import { Glyphicon } from 'react-bootstrap'
 import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd'
 import cx from 'classnames'
+import _ from 'lodash';
```

Comment:
i've started importing lodash like this:
`import { chunk } from 'lodash'`
I think it makes the bundle smaller. That's what I'm going for anyway

Reply:
Noted.  Will do in the future.
(I actually searched the project to find the convention and found that it was being done this way.  I'll port any others that I find in the future).